include/internal/hpke_util.h

```diff
@@ -15,7 +15,6 @@
 # define OSSL_HPKE_MAX_SECRET 64
 # define OSSL_HPKE_MAX_PUBLIC 133
 # define OSSL_HPKE_MAX_PRIVATE 66
-# define OSSL_HPKE_MAX_NONCE 12
```

Comment:
Where has this gone?

Reply:
wasn't used in the end, the other one was (``OSSL_HPKE_MAX_NONCELEN``) which is still there